llama-index-integrations/vector_stores/llama-index-vector-stores-duckdb/llama_index/vector_stores/duckdb/base.py

```diff
@@ -396,11 +392,10 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
             node = TextNode(
                 id_=_row[0],
                 text=_row[1],
-                embedding=_row[2],
-                metadata=json.loads(_row[3]),
+                metadata=json.loads(_row[2]),
```

Comment:
Why did we remove embedding from the returned node? This would be a breaking change for some users who are expecting it